types/btoa/btoa-tests.ts

```diff
@@ -1,3 +1,6 @@
 import btoa = require('btoa');
+import fs = require('fs');
 
 btoa('foo');
+
+btoa(fs.readFileSync('foo.css'));
```

Comment:
Might be simpler to just use `Buffer.from` instead of going through `fs`.

Reply:
Right, replaced by `Buffer.from('foo')`